config/sync/core.extension.yml

```diff
@@ -44,7 +43,6 @@ module:
   layout_discovery: 0
   link: 0
   locale: 0
-  memcache: 0
```

Comment:
Don't we use memcache anymore?

Reply:
The purpose is to move it out of the config flow. So it can be deactivated during local devlopment by running pm:uninstall

Excluding it in settings.php ensures that it will not affect stg or prod (Or local before manual deactivation)

"When syncing configuration, if an excluded module is already installed, it will not be uninstalled by the configuration synchronization, and dependent configuration will remain intact."

Reply:
Another approaach could be to add $settings['config_exclude_modules'] to settings.local.php . I think the result would be pretty much the same. Differende would be the extensions.yml file in github, whether memcahe by default is activated or deactivated.

Reply:
I don't know if we want to use `memcache` in production or not – we use it somewhere but not everywhere (cf. https://github.com/search?q=org%3Aitk-dev+memcache+language%3AYAML+path%3A**%2Fcore.extension.yml&type=code).

But if we want to disable it during development we should document how to do that rather than removing the module from the production setup. Why do we want to uninstall it during development? Can't we just disable the cache on `/admin/config/development/settings`?